app/Models/Signup.php

```diff
@@ -31,7 +31,7 @@ class Signup extends Model
      *
      * @var array
      */
-    protected $fillable = ['id', 'northstar_id', 'campaign_id', 'quantity', 'quantity_pending', 'why_participated', 'source', 'source_details', 'details', 'created_at', 'updated_at'];
+    protected $fillable = ['id', 'northstar_id', 'campaign_id', 'why_participated', 'source', 'source_details', 'details', 'created_at', 'updated_at'];
```

Comment:
Should `quantity` be fillable still?

Reply:
Nope – we don't want anyone to set quantity on the post by hand (only via the model event).

Reply:
Did you mean on the signup?

Reply:
Oh yep - sorry! 🤦‍♂ We do want folks to set quantity on the posts, but the field on the signup should not be mass-assignable & only updated by this model event.